src/commands/sbom/upload.ts

```diff
@@ -148,7 +143,7 @@ export class UploadSbomCommand extends Command {
     }
 
     const uploadTimeMs = (Date.now() - startTimeMs) / 1000
-    this.context.stdout.write(renderSuccessfulCommand(this.basePaths.length, uploadTimeMs))
+    this.context.stdout.write(renderSuccessfulCommand(this.basePath.length, uploadTimeMs))
```

Comment:
`this.basePath.length` would be the length of the string 😬

Reply:
You probably need to rewrite a little the `renderSuccessfulCommand()` function too

Reply:
@juli1 I'm not sure why you marked this as resolved? 🤔

Reply:
The commit was still pending on my laptop! Should be resolved now for real!

Reply:
oh, sorry 😁